csharp/src/Microsoft.ML.OnnxRuntime/SessionOptions.shared.cs

```diff
@@ -402,12 +408,9 @@ public void AppendExecutionProvider(string providerName, Dictionary<string, stri
         [ObsoleteAttribute("RegisterCustomOpLibrary(...) is obsolete. Use RegisterCustomOpLibraryV2(...) instead.", false)]
         public void RegisterCustomOpLibrary(string libraryPath)
         {
-            IntPtr libraryHandle = IntPtr.Zero;
-            var libraryPathPinned = GCHandle.Alloc(NativeOnnxValueHelper.StringToZeroTerminatedUtf8(libraryPath), GCHandleType.Pinned);
-            using (var pinnedlibraryPath = new PinnedGCHandle(libraryPathPinned))
-            {
-                NativeApiStatus.VerifySuccess(NativeMethods.OrtRegisterCustomOpsLibrary(handle, pinnedlibraryPath.Pointer, out libraryHandle));
-            }
+            var utf8Path = NativeOnnxValueHelper.StringToZeroTerminatedUtf8(libraryPath);
+            // The handle is leaking in this version
```

Comment:
Do we plan to address this?

Reply:
No, the API is public, it is just obsolete for the reason.